Userland/Demos/Mandelbrot/Mandelbrot.cpp

```diff
@@ -118,8 +121,12 @@ class MandelbrotSet {
     void calculate_pixel(int px, int py, int max_iterations)
     {
         auto iterations = mandelbrot(px, py, max_iterations);
-        auto whole_iterations = floor(iterations);
-        auto partial_iterations = fmod(iterations, 1);
+        if (iterations == max_iterations) {
+            m_bitmap->set_pixel(px, py, Color::Black);
+            return;
+        }
+        auto whole_iterations = static_cast<int>(iterations);
+        auto partial_iterations = iterations - whole_iterations;
```

Comment:
Why did you change these two lines?

Reply:
Before i figured out the "only calculate set in paint event" thing the julia set
rendering was way too slow. So this is left over from me trying to optimize
stuff. Is cast-to-int actually faster than `floor(float)`?

removed for now.

Reply:
They do different things
Cast to int gives you an int and truncates, and integer math is in 80% of cases faster than floating point math, if you stay with integer math
Floor, floors (duh) and you still have a float